### PR TITLE
Kilkun-balance episode 4: The Corporations Strike Back Part 1 of 2

### PR DESCRIPTION
**Pull request by kilkun**
This PR is the beginning of the Corporate death commandos being put back on the pedestal they deserve to be on. No longer are mild mannered crewmembers a match for these badasses. No, even the Space Wizard's federation will be praying to the honkmother should these progrom soldiers be sent out against them.

Full changes:

> Created new species datum: The Corporate Post Human
> syndicate
> These dead eyed cyborgs are the state of the art. Cybernetic limbs grant them a ridiculous strength boost, dealing up to 50 damage, with grazing hits doing an incredibly brutal 20 damage.
> Armor plating and chemically treated skin have rendered them pale and without hair, but the upsides are that their hide is extremely tough against all forms of damage. Think that plasmafire will protect you from the wrath of your corporate overlords? Think again! They'll march through unphazed to slam your head repeatedly into the flooring of your station, whether you are loyal or not! Amazing!
> Mechanical nutrition systems keep the agents fed and healthy, meaning they never have to eat. Which is good, because we also grafted a mask over their mouth.

Included in every Nanotrasen Death commando is now a military grade adrenal implant! Unlike the civilian, cheap version given to traitors, this special device utilizes state-of-the-art bluespace technology developed in tandem with Nanotrasen and Cybersun Industries to create the ultimate combat implant. This highly experimental device transforms whatever lucky highly-illegal commando was selected to be fitted into a nigh unstoppable death machine!

The result is that once more, corporations are back on top of the food chain. Or at least halfway up it. There's more to come in Episode 5, so make sure not to miss that!

Space Jesus bless Nanotrasen and it's fine employees with these watchful souls looking out for us!
post_human
[Inspired by Syndicate](https://www.youtube.com/watch?v=bCMpRNZojA4)

### CODE REVIEW

**Comment by PKPenguin321** [comment]
neat

**Comment by DaedalusGame** [comment]
>Part 1 out of 2 :scream:

File: code/game/objects/items/weapons/implants/implant_misc.dm [hunk 1]
```diff
@@ -48,6 +48,24 @@
 	imp_in.reagents.add_reagent("omnizine", 10)
 	imp_in.reagents.add_reagent("stimulants", 10)
 
+/obj/item/weapon/implant/adrenalin/deathsquad
+	name = "adrenal implant"
+	desc = "Removes all stuns and knockdowns."
+	icon_state = "adrenal"
+	origin_tech = "bluespace=4;biotech=10;combat=10;syndicate=4"
+	uses = 300
```
Comment (Shadowlight213) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: You have 300 uses set here, yet the description says over 500.

**Comment by kilkun** [comment]
Oh yes. Where we are going, there will be no **gods** and no kings.

Only _The Company._

File: code/game/objects/items/weapons/implants/implant_misc.dm (hunk 1 above)
Comment (kilkun) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: @Shadowlight213 I don't think there's ever going to be a time where it actually gets used more than 50x in a round considering you OD on omnizine and synaptizine pretty quick. Maybe 200 uses got burned up in training.

**Comment by KorPhaeron** [comment]
You know how in the fluff Space Marines are described in completely absurd ways? Elite, the best, nothing can stop em, shrug off bullets, etc etc

But then in 40k tabletop and videogames and even stories they die in droves?

It's because while coming up with a super badass might be fun, they're actually rather boring to play as/against, and it makes for a dull story if the outcome is predetermined and there wasn't even the illusion of tension or drama. You could have just set GODMODE on these guys for the same effect.

I also don't really like this for LORE reasons because nanotrasen having immortal people doesn't really seem to make sense but I guess the lore isn't really a thing anyway.

Probably doesn't matter either way since death squad only ever get spawned on the shuttle dock post round now, and they will definitely never see the station in this form.

I'm curious what part 2 is though.

**Comment by DaedalusGame** [comment]
nuke implants that blow up when a death squad member is killed

**Comment by Shadowlight213** [comment]
I was actually gonna say give singularity implants that make a momentary stage 2 singulo on death

**Comment by DaedalusGame** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=32

**Comment by kilkun** [comment]
@KorPhaeron There was a point in SS13 history, a very, very long time ago that this is a callback to.

Back then Deathsquad wasn't called deathsquad. They weren't even called death commandos. They were called SWAT. Their armor had a 90% chance to block all damage from most types of attacks, even the dreaded revolver which they wielded with their energy swords. They had hand teleporters and thermals and when they showed up shit had hit the fan. Not even the Syndicate SWAT were a match.

This is what I'm elevating them to. They are the final boss of SS13. You wanna prove you are robust? You've gone toe to toe with the Head of Security and smashed apart his department? You killed all the heads? You're a real human SAMelonDude and killed everybody on the station as traitor? Deadchat all chants in unison for it. For _THEM_. And you know what? An admin actually pushes the button the absolute madman.

This it the chance for them to prove themself. A rite to passage that they are as good as the Melon man and a worthy successor of the art of griff.

That is what Deathsquad should be. The final boss. Not a cheap joke antag that gets disarmed and toolboxed 5 byond ticks after their shuttle docks.

Part 2 is about making Deathsquad the hard counter to Ascended shadowlings among a myriad of other features I didn't put in. Including the other side to Deathsquad's coin and probably the dustiest button on the admin panel that is about to get a whole lot dustier.

**Comment by PKPenguin321** [comment]
If we can keep these separate from the current deathsquad that'd be cool

**Comment by kilkun** [comment]
Why when deathsquad never gets used anyways?

It's the end game admin button. This makes the end game come faster, and of course reasserts the corporations as the biggest power in the game, instead of faggy shadowlings or space bums with plasma stained beards and stanky ass robes.

**Comment by kevinz000** [comment]
Yeaaah no, keep em separated. Those 5 deathsqaud guys are killing everyone. They're having fun. But everyone else? Not so much. The game's about having fun and taking everyon'e CHANCE of fighting back is not fun.

**Comment by Sarcalogo** [comment]
Said by Mekhi the telearmor abusing faggot.

**Comment by newfren** [comment]
Using the tele armour makes your argument invalid now :ok:

**Comment by Alek2ander** [comment]
`You are becoming hysterical`

**Comment by Supermichael777** [comment]
Use the sec ert as the current stoppable purge team. if admins want a stoppable response they get the short straw team rather than DEATH-COMANDOS. anyway death-squads are explicitly burn the evidence types while one out of control guy/ ailums really just needs sec ert.

**Comment by RemieRichards** [comment]
@kilkun While you may not want to, keeping the current deathsquad and your super deathsquad separated will increase the chance the PR will actually get merged (Both my opinion, and a few of the players here have also suggested the split)

**Comment by Shadowlight213** [comment]
@Supermichael777 Rather than making all new players arrive as deathsquad, I think having a notification on the pick job to join as screen, similar to the "the station is undergoing emergency evacuation procedures" notice that informs that the station is currently being purged and it would be a VERY bad idea to join. If they ignore the warning then it's their loss.
I'm worried that an admin might give the deathsquad a briefing that a new player just joining would miss and screw everything up.

**Comment by Shapsy** [comment]
wait since when are deathsquads "laughable easy to defeat", every time I've seen them spawned they've wrecked shit (except for one round where everyone was a ling and even then they managed to down most of the lings at least once or twice)

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -45,6 +45,30 @@ datum/species/human/spec_death(gibbed, mob/living/carbon/human/H)
...
+	need_nutrition = 0//They don't need to eat
+	say_mod = "asserts"//Their word is law
+	speedmod = 1//The cybernetic limbs cost a fortune
+	brutemod = 0.5//and it cost them their humanity
+	burnmod = 0.5//but they were worth everything
+	coldmod = 0
+	heatmod = 0//they can walk through a plasmafire, naked, unphased
+	punchdamagelow = 20
+	punchdamagehigh = 50//they are inhumanly strong
+	punchstunthreshold = 100
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types.dm]: this means they'll never stun, it only happens if a punch does damage equal to or above this value

**Comment by InsaneHyena** [comment]
Hi.

Please explain to me why this exists. Imagine a situation - you are a shadowling. You've worked hard to gather 15 thralls and ascend, when your team mates die pointlessly, because they forgot that sun gives light. Finally, you ascend... And some deashquad jerkoff with unlimited adrenal, superweapons and "hard counter to Ascended shadowlings" gun appears, and then kills you. All your effort through the round is now rendered absolutely worthless, because somebody decided that "let's make NanoTrasen literally all-powerful god corporation with immortal juggernauts as soldiers" is a good piece of lore or something.

File: code/modules/mob/living/carbon/human/species_types.dm (hunk 2 above)
Comment (kilkun) [inline code/modules/mob/living/carbon/human/species_types.dm]: That's kind of the point, they don't really NEED to stun when their fists are basically short range pulse rifles with a no ammo.

**Comment by kilkun** [comment]
@InsaneHyena Because it's not like you ascend and deathsquad is instantly called.

If you are a big enough shitter to get the admins to actually push the Deathsquad button you absolutely 100% deserve your fate. Also 15 thralls is pretty easy to take if you don't have assistants descending upon you in a tide with dual wielded torches and glowshrooms.

And besides that if you actually manage TO get a deathsquad to spawn against you, chances are they aren't going on the station and will just wait for you at the CENTCOMM docks. At which point it doesn't matter if your special snowflake sling gets gibbed.

@RemieRichards I like the idea suggested up that Sec ERT takes the role of flukesquad. Deathsquad should be reserved to when the round is completely and utterly fucked and it doesn't even matter if the antags get their shit pushed in.

**Comment by Shapsy** [comment]
>Deathsquad should be reserved to when the round is completely and utterly fucked and it doesn't even matter if the antags get their shit pushed in.

...you mean exactly like they are right now?

**Comment by hornygranny** [comment]
@kilkun ascending forcecalls the shuttle, there isn't a situation in which that would be required

**Comment by kilkun** [comment]
@Shapsy yea, but any robust antag with a taser/stunbaton can take out your average deathsquad, ESPECIALLY if they split up.

@hornygranny just because the shuttle is called doesn't mean deathsquad can't get called too.

**Comment by optimumtact** [comment]
Another pointless feature that will add more guns and more uniforms that will probably never get used unless they get dragged into the game.

**Comment by kilkun** [comment]
A subtype of the pulse rifle is planned that will be excluded from things like summon guns and other such stuff, and no uniforms have been added or are getting added. not sure what you are talking about.

**Comment by Iamgoofball** [comment]
why dont we just give deathsquaddies macrobomb implants and call it a day

**Comment by InsaneHyena** [comment]
So, an actual conversation with actual SS13 players was had in the OOC chat. Kilkun kept pushing this abomination, constantly alternating between "deathsquad never gets called anyway, you won't even notice the changes" and "deathsquad gets dunked, which is wrong, after my changes deathsquad will never get dunked again". Eventually Kor made a poll about it. Here are the results - https://i.gyazo.com/d332ef21e962fb101d42fca3d4a6a0f8.png

**Comment by kilkun** [comment]
Like I said, most players don't care. Missing the forest for the trees here to be honest.

And hey at the end of the day it's a new species datum. Deathsquad can always be reverted back to normal by deleting literally 3 lines.

**Comment by MMMiracles** [comment]
If it never gets used then whats the point of changing it so there's even more reason it'll not get used?

Literally safety-padding every possible corner on an already padded position. If you get dunked as a death squad either you were fighting something literally unkillable or you really, really suck at the game.

**Comment by AdamElTablawy** [comment]
Dumb, no.

The poll said they didn't want it those who cared, and the majority of people who don't care isn't an excuse.

This is just stupid. Just set them to godmode if you're that desparate-Deathsquad was meant to be end round GG fuck you, but you could kill them if you tried sufficiently hard enough.

They won't get used AT ALL with this change, so don't bother.

**Comment by Core0verload** [comment]
Fuck no. We don't need any unstoppable bullshit antags.

**Comment by Jazaen** [comment]
> the hard counter to Ascended shadowlings

Huh, and how do you want to do THAT?

**Comment by Jazaen** [comment]
Though the thing itself is pretty rad, since right now thanks to our lovely TG-brand stun combat one taser can wreck their shit. And I see no point in objecting, since it's not like this will make them used less.
Maybe just add a fifth category in the "Call Help" admin button, labelled "Extermination" (or something grandiose, I don't know), to send these guys instead of your normal Deathsquad.

File: code/game/objects/items/weapons/implants/implant_misc.dm [hunk 3]
```diff
@@ -48,6 +48,24 @@
...
+
+/obj/item/weapon/implant/adrenalin/deathsquad/get_data()
+	var/dat = {"<b>Implant Specifications:</b><BR>
+				<b>Name:</b> Cybersun Industries Military Grade Adrenaline Implant<BR>
```
Comment (Cheridan) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: >NT death squad troops using syndicate tech

Comment (PKPenguin321) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: It's a conspiracy

**Comment by DrCelt** [comment]
That's all well and good but when are we getting https://github.com/d3athrow/vgstation13/pull/5685 ?

**Comment by Shadowlight213** [comment]
@drcelt wrong codebase?
This is tg. That's vg.

File: code/game/objects/items/weapons/implants/implant_misc.dm (hunk 3 above)
Comment (kilkun) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: Remember how the syndicate is secretive and that corporations aren't publicly affiliated with it.

It's actually kind of a hint about deathsquad's counter WHO I HAVE BEEN INFORMED WAS REMOVED FROM THE CODEBASE.

**Comment by RemieRichards** [comment]
@Shadowlight213 compare the authors of both PRs
(Hint, they're the same person)

**Comment by Shadowlight213** [comment]
THE SEPERATION BETWEEN THE CODEBASES IS ABSOLUTE.

File: code/game/objects/items/weapons/implants/implant_misc.dm (hunk 3 above)
Comment (Shadowlight213) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: Ninja isn't removed. It just got most of its features gutted and the objective removed.
It's even more of a hard counter to DS currently, as thermals don't reveal the ninja anymore and the ninja stars completely bypass the ds armor.

**Comment by kilkun** [comment]
@Shadowlight213 Celt is the type of funposter who would love to cross the streams. also he's loosely 'associated' with me.

@DrCelt I said before: In a long ways away. I did this as a fun little jaunt to get away from GML and it was pretty simple to do. The ninja recode will take substantially more time, effort and knowledge of DML to complete. Maybe I'll work on it some more after I'm done with my kickstarter and am not crunching to finish a demo.

good to see you btw bb hope your snow station 13 is coming along good.

**Comment by kilkun** [comment]
@Core0verload Deathsquad aren't really conventional antags. Are you the kind of person who believes nar-sie should also be killable?

@Jazaen Deathsquad's special pulse rifle introduced in part 2 will gib targets not wearing armor and simplemobs alike. Ascendeds fall into this category. Darkmatter > regular matter.

Also it's gesture will bounce off of the deathsquaddies because corporate power > natural power. It's science I ain't gotta explain shit.

File: code/game/objects/items/weapons/implants/implant_misc.dm (hunk 3 above)
Comment (kilkun) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: Not Ninja, Syndicate Black Ops.

The Syndicate boys with pulse rifles, silenced .45s, black softsuits maulers and their own stealth pod. When part 2 comes around they're coming back with a fresh new skin and all the feats Deathsquad has.

**Comment by Core0verload** [comment]
>Are you the kind of person who believes nar-sie should also be killable?

I actually killed one, back, when it was possible to one shot the singulo with a bomb. It took two 14-14-14 exploit bombs - two because my timing was wrong with a first one.

**Comment by Jazaen** [comment]
Wait. The ascended shadowling is pretty much the only thing onboard that nothing can kill. And you are basically making DS immune to it's normal mode of attack. And you are giving them a cannon that gibs simple mobs (this also includes all cult constructs, and possibly some new bosses if anyone cares to add them) outright?

**Comment by kilkun** [comment]
traitors, shitty revs, gangs and nuke ops stand no chance against a deathsquad. why should slings?

**Comment by MMMiracles** [comment]
@kilkun Because end-round slings are the result of them completing all of their objectives and ascending, similar to a cult summoning nar-sie.

**Comment by kilkun** [comment]
and deathsquad is the admin's 'go fuck yourself' button and there's no reason for any player controlled anything to be able to stand up to them.

Unless it's a stupidly robust traitor or a competent wizard.

**Comment by MMMiracles** [comment]
An ascended sling can literally gib people with its mind at any distance in an instant are you really saying something should exist that's stronger/capable of defeating that.

**Comment by kilkun** [comment]
Wizards already can (and for good reason) and if Ninjas still gibbed with phase jaunt they would be capable of killing asslings too. And a deathsquad is just as much of a death sentence to the normal crew as an ascended sling is even before this update.

And yes absolutely it should. Not even asslings should be immune from the admin 'fuck you'. Deathsquad should be seen as the more interactive alternative to just pressing the "detonate nuke" button, one that gets 5 lucky players to vent their frustrations on the shitty round that actually managed to get an admin to spawn one in the first place.

Both from a lore perspective that the corporations are bigger than gods and from a practical perspective. It's not like this has any effect on 99.99% of rounds and if/when it gets test merged you might find out it doesn't result in that much bigger of a difference. Other than it's going to take more than a taser to bring down the largest corporation's most elite and feared group of highly trained super commandos.

**Comment by Cheridan** [comment]
So I'm reading through and I'm not really seeing anyone supporting this. Like, the sprite and the fluff with the unique mutantrace is pretty cool, but mechanically they're just too strong. The few people who enjoy when a Deathsquad comes, they like it BECAUSE there's a fight. The few people who have survived the chaos that lead up to the point aren't going to enjoy godmode pulseriflehands men steamrolling them.

**Comment by duncathan** [comment]
This guy did his work on master

**Comment by kilkun** [comment]
because branches are for nerds.

**Comment by kilkun** [comment]
The species datum will return.

At some point.

Likely as a single-antag gametype, similar to wizard, with some nerfs here and there in the DR department along with some extra abilities like punching through walls and perma thermals.

**Pull request closed without merging**